src/runtime/objmodel.cpp

```diff
@@ -1449,6 +1460,13 @@ extern "C" Box* getattr(Box* obj, const char* attr) {
     static StatCounter slowpath_getattr("slowpath_getattr");
     slowpath_getattr.log();
 
+    if (obj->cls->tp_getattr) {
+        Box* rtn = obj->cls->tp_getattr(obj, const_cast<char*>(attr));
+        if (rtn == NULL)
+            raiseExcHelper(AttributeError, "%s", attr);
```

Comment:
yup.  in the case of selectmodule, Py_FindMethod will have installed AttributeError as the thread's exception.  fixing.